core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -3762,7 +3766,7 @@ public Object getDynamic(String token) {
     public synchronized void doConfigSubmit( StaplerRequest req, StaplerResponse rsp ) throws IOException, ServletException, FormException {
         BulkChange bc = new BulkChange(this);
         try {
-            checkPermission(ADMINISTER);
+            checkPermission(MANAGE);
```

Comment:
FTR The "protection" against illegal modifications is done by filtering out the descriptor with `getSortedDescriptorsForGlobalConfigUnclassified`

Reply:
Most of the explanation in https://github.com/jenkinsci/jenkins/pull/4483#issuecomment-585896585 still applies.